lib/types/db.ts

```diff
@@ -25,6 +26,17 @@ export type CurrencyAttributes = CurrencyFactory & {
 
 export type CurrencyInstance = CurrencyAttributes & Sequelize.Instance<CurrencyAttributes>;
 
+// SwapDeals
+export type SwapDealFactory = SwapDeal;
```

Comment:
I believe we want everything in `SwapDeal` *except* `makerToTakerRoutes` for now. You can do this with `type SwapDealFactory = Pick<SwapDeal, Exclude<keyof SwapDeal, 'makerToTakerRoutes'>>;`